src/System.Management.Automation/utils/EncodingUtils.cs

```diff
@@ -25,13 +26,17 @@ internal static class EncodingConversion
         internal const string Default = "default";
         internal const string OEM = "oem";
         internal static readonly string[] TabCompletionResults = {
-                Ascii, BigEndianUnicode, OEM, Unicode, Utf7, Utf8, Utf8Bom, Utf8NoBom, Utf32
+                Ascii, BigEndianUnicode, OEM, Unicode, Utf7, Utf8, Utf8Bom, Utf8NoBom, BigEndianUtf32, Utf32
             };
 
+        // BigEndianUTF32 encoding is possible, but requires creation
+        internal static Encoding BigEndianUTF32Encoding = new UTF32Encoding(bigEndian: true, byteOrderMark: true);
```

Comment:
Should this field be `readonly`?

Reply:
Originally I had just copied this from another function where it was defined like this, but I updated this commit to just embed it in the dictionary.  Since we're only using it once, no need to generate a separate object.